### ISSUE TITLE
Signup: site-type options jump when selected

### ISSUE THREAD

**Issue by shaunandrews** [Automattic/wp-calypso#29292]
When first selecting the Business or Online store options, the height of the row changes causes the layout to jump a bit. Checkout the GIF to see it in action:

![jumpy jump](https://user-images.githubusercontent.com/191598/49751018-5fe73e00-fc7a-11e8-9405-59582e4d40b3.gif)

**Comment by shaunandrews** [comment]
Seems to be in Safari, but not in Chrome.

**Comment by arunsathiya** [comment]
Noting that I tested this as well, when I had to create a new WordPress.com account, but I am not able to reproduce the same behaviour mentioned in the original report/GIF. My test was on Safari Version 12.0.1 (14606.2.104.1.1) for macOS 10.14.1 - Mac OS Mojave.

[Video showing that behaviour](https://cloudup.com/c4NupGS9OMU).

At one point of time though, I did see `Blog` choice show the jump. The same can be 00:06 on the video.
